lib/voicemail/plugin.rb

```diff
@@ -12,6 +12,8 @@ class Plugin < Adhearsion::Plugin
       menu_tries 3, desc: "Tries to get matching input for all menus"
       datetime_format "Q 'digits/at' IMp", desc: "Format to use for message date and time formatting"
 
+      go_to_menu_timeout 5, desc: "Timeout used on the go to menu digit"
```

Comment:
This description could be more clear, something like: "Timeout for the mailbox greeting input, after which it will proceed to the menu"